metadata-ingestion/src/datahub/ingestion/source/kafka_connect.py

```diff
@@ -1199,6 +1476,9 @@ def construct_job_workunits(
                     aspect=models.DataJobInputOutputClass(
                         inputDatasets=inlets,
                         outputDatasets=outlets,
+                        inputDatasetFields=inlets_fields,
```

Comment:
What is the significance of inputDatasetFields, outputDatasetFields ? Looks like everything is already covered in fineGrainedLineages.